libmscore/fraction.h

```diff
@@ -188,20 +188,26 @@ class Fraction {
             return *this;
             }
 
-      #if 0
+
       Fraction& operator/=(int val)
             {
             _denominator *= val;
+            if (_denominator < 0) {
```

Comment:
Is it really necessary in the PR where you just improve tooltips?

Reply:
I've added 
https://github.com/musescore/MuseScore/pull/6080